index.src.html

```diff
@@ -3317,27 +3317,250 @@
+  four top-left pixels. The [=sub-sampling factor=] is 2 in both the horizontal
+  and vertical direction.
+</div>
+
 <dl>
   <dt><dfn enum-value for=PixelFormat>I420</dfn></dt>
   <dd>
-    Planar 4:2:0 YUV.
+    This format is composed of three distinct planes, one plane of Luma and two
+    planes of Chroma, denoted Y, U and V, and present in this order. It is also
+    often refered to as Planar YUV 4:2:0.
+
+    The U an V planes are [=sub-sampled=] horizontaly and vertically by a
+    [=factor=] of 2 compared to the Y planes.
+
+    Each sample in this format is 8 bits.
+
+    There are {{VideoFrame/codedWidth}} * {{VideoFrame/codedHeight}} samples
+    (and therefore bytes) in the Y plane, arranged starting at the top left in
+    the image, in {{VideoFrame/codedHeight}} lines of {{VideoFrame/codedWidth}}
+    samples.
+
+    There is {{VideoFrame/codedWidth}} * {{VideoFrame/codedHeight}} / 4 samples
+    (and therefore bytes) in the two U and V planes, arranged starting at the
+    top left in the image, in {{VideoFrame/codedHeight}} / 2 lines of
+    {{VideoFrame/codedWidth}} / 2 samples.
+
+    The {{VideoFrame/codedWidth}} and {{VideoFrame/codedHeight}} MUST be even.
+    Similarly, the visible rectangle ({{VideoFrame/displayWidth}} and
+    {{VideoFrame/displayHeight}}) MUST be even.
   </dd>
-</dl>
+  <dt><dfn enum-value for=PixelFormat>I420A</dfn></dt>
+  <dd>
+
+    This format is composed of four distinct planes, one plane of Luma, two
+    planes of Chroma, denoted Y, U and V, and one place of alpha values, all
+    present in this order. It is also often refered to as Planar YUV 4:2:0 with
+    an alpha channel.
+
+    The U an V planes are [=sub-sampled=] horizontaly and vertically by a
+    [=factor=] of 2 compared to the Y and Alpha planes.
+
+    Each sample in this format is 8 bits.
+
+    There are {{VideoFrame/codedWidth}} * {{VideoFrame/codedHeight}} samples (and
+    therefore bytes) in the Y and alpha plane, arranged starting at the top left
+    in the image, in {{VideoFrame/codedHeight}} lines of
+    {{VideoFrame/codedWidth}} samples.
+
+    There are {{VideoFrame/codedWidth}} * {{VideoFrame/codedHeight}} / 4 samples
+    (and therefore bytes) in the two U and V planes, arranged starting at the
+    top left in the image, in {{VideoFrame/codedHeight}} / 2 lines of
+    {{VideoFrame/codedWidth}} / 2 samples.
+
+    The {{VideoFrame/codedWidth}} and {{VideoFrame/codedHeight}} MUST be even.
+    Similarly, the visible rectangle ({{VideoFrame/displayWidth}} and
+    {{VideoFrame/displayHeight}}) MUST be even.
+  </dd>
+  <dt><dfn enum-value for=PixelFormat>I422</dfn></dt>
+  <dd>
+
+    This format is composed of three distinct planes, one plane of Luma and two
+    planes of Chroma, denoted Y, U and V, and present in this order. It is also
+    often refered to as Planar YUV 4:2:2.
+
+    The U an V planes are [=sub-sampled=] horizontaly by a [=factor=] of 2
```

Comment:
Here and below, I would make explicit mention of the cases where the factor is 1.